source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,79 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const shortcutClass = new Map<string, string>([
+	['e', '.rgh-edit-comment, [aria-label="Edit comment"]'],
+	['d', '[aria-label="Delete comment"]'],
+	['h', '[aria-label="Hide comment"]'],
+	['j', ''],
+	['k', '']
```

Comment:
These can be dropped and moved exactly where needed (1/4)

```suggestion
	['h', '[aria-label="Hide comment"]']
```